tests/components/airly/test_config_flow.py

```diff
@@ -0,0 +1,92 @@
+"""Define tests for the Airly config flow."""
+import json
+
+from airly.exceptions import AirlyError
+from asynctest import patch
+from tests.common import load_fixture, MockConfigEntry
+
+from homeassistant import data_entry_flow
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+from homeassistant.components.airly import config_flow
+from homeassistant.components.airly.const import DOMAIN
+
+CONFIG = {
+    CONF_NAME: "abcd",
+    CONF_API_KEY: "foo",
+    CONF_LATITUDE: 123,
+    CONF_LONGITUDE: 456,
+}
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    flow = config_flow.AirlyFlowHandler()
+    flow.hass = hass
+
+    result = await flow.async_step_user(user_input=None)
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
+
+
+async def test_invalid_api_key(hass):
+    """Test that errors are shown when API key is invalid."""
+    with patch(
+        "airly._private._RequestsHandler" ".get",
```

Comment:
Please remove the split in the string. That's a side effect after black formatting.